app/assets/javascripts/snowcrash.js

```diff
@@ -23,6 +23,7 @@
 //= require snowcrash/modules/configurations
 //= require snowcrash/modules/console_updater
 //= require snowcrash/modules/export
+//= require snowcrash/modules/index_table
```

Comment:
something more descriptive maybe? js table, table extender, "index" is confusing as it could mean the MVC view.

Reply:
I would go with `items_table`. Yes?